parsec/core/fs/remote_loader.py

```diff
@@ -98,7 +128,9 @@ async def _load_realm_role_certificates(self, realm_id: Optional[EntryID] = None
 
             # Now verify each certif
             for unsecure_certif, raw_certif in unsecure_certifs:
-                author = await self.remote_device_manager.get_device(unsecure_certif.author)
+
+                with translate_remote_device_manager_errors():
+                    author = await self.remote_device_manager.get_device(unsecure_certif.author)
```

Comment:
`self.remote_device_manager.get_user` is also used in `WorkspaceFS.get_reencryption_need`, maybe we should improve exception handling there too ?

Reply:
a518ebd